x/axelarnet/types/genesis.go

```diff
@@ -25,10 +26,10 @@ func DefaultGenesisState() *GenesisState {
 		Params:           DefaultParams(),
 		CollectorAddress: nil,
 		Chains: []CosmosChain{{
-			Name:       ModuleName,
+			Name:       exported.Axelarnet.Name,
```

Comment:
question: does it break the existing code?

Reply:
better to use lowercase

Reply:
There were some validation checks that were too strict. I fixed those. I don't think it should be an issue other than that. We can consider changing `axelarnet` to `Axelarnet` in the genesis file.